client/electron/electron-builder.json

```diff
@@ -32,30 +34,32 @@
     "files": [
       "client/electron/linux_proxy_controller/dist",
       "client/electron/icons/png",
-      "client/output/build/linux",
-      "!client/output/build/linux/*.h"
+      "output/client/linux-amd64",
```

Comment:
Also this will break the app because we rely on the paths:

https://github.com/Jigsaw-Code/outline-apps/blob/c8864f1d7c8e16b26a9aef29cb66794e0cdf113f/client/electron/app_paths.ts#L48-L68

Reply:
Thanks for the catch. I'm sending a fix in https://github.com/Jigsaw-Code/outline-apps/pull/2513.

But this is bad practice... We shouldn't have our build directory structured reflected in our built app. This should be something like `path.join(unpackedAppPath(), 'tun2socks')`.